app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,9 @@
   </head>
 
   <body>
+    <nav class="topnav">
```

Comment:
Anything in this application.html file will be loaded before anything else. Every page will have this